packages/pedantic/lib.js

```diff
@@ -0,0 +1,122 @@
+const path = require('path');
+const chalk = require('chalk');
+const { promises: fs } = require('fs');
+const { default: sortImports } = require('import-sort');
+const { debuglog } = require('util');
+const prettier = require('prettier');
+const { getConfig } = require('import-sort-config');
+
+const ConsoleUtilities = require('@4c/cli-core/ConsoleUtilities');
+const ArgUtilities = require('@4c/cli-core/ArgUtilities');
+
+const debug = debuglog('pedantic');
+
+function sortFileImports(content, filePath) {
```

Comment:
this should only run on JS, no? will it just ignore all non-JS silently?

Reply:
it'll ignore since there won'der be a config setup anywhere for non-js extensions. I will probably also manually ignore `.d.ts` files since they can have lots of non-top level imports and import-sort has no idea what to do there